assets/charts/control-plane/calico-host-protection/templates/psp.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
I think we could make this policy more strict. Start with restricted policy and then allow what's required.